apps/rush/src/utilities/PackageChangeAnalyzer.ts

```diff
@@ -83,13 +83,46 @@ export class PackageChangeAnalyzer {
       }
     });
 
+    /* Incremental Build notes:
+     *
+     * Temporarily revert below code in favor of replacing this solution with something more
+     * flexible. Idea is essentially that we should have `gulp-core-build` (or other build tool)
```

Comment:
No, I copied this from a markdown comment in a VSTS PR. I can remove it, though.

---
In reply to: [126800243](https://github.com/Microsoft/web-build-tools/pull/269#discussion_r126800243) [](ancestors = 126800243)